dotnet/test/AutoGen.SourceGenerator.Tests/ApprovalTests/FunctionCallTemplateTests.TestFunctionCallTemplate.approved.txt

```diff
@@ -1,4 +1,4 @@
-﻿﻿//----------------------
+﻿//----------------------
```

Comment:
Do you still need to keep this .approved.txt?

Reply:
I hope so. I think the .approved.txt file serves as the baseline for what is considered correct output when running the tests? Please correct me if I am wrong. Thanks!

Reply:
The `approval.txt` is automatically created by [`ApprovalTests`](https://github.com/approvals/ApprovalTests.Net) when you call `Approvals.Verify(xxx)`. Since the new test doesn't call `Approvals.Verify` you probably can safely delete that `approved.txt` file.

Reply:
Okay! Thank you very much for this information. I will delete it!

Reply:
@prithvi2226 Oh, I am super super sorry. I thought the `.approval.txt` file is newly added by the tests from this PR, but just find out that it's not. Could you add it back. Again, I'm really really sorry for the misguidance.

Reply:
It's completely fine. There is no problem I can add it again. Thanks for the quick response!